config.md

```diff
@@ -463,6 +449,20 @@ Here is a full example `config.json` for reference.
                 "gid": 0
             }
         ],
+        "uidMappings": [
+            {
+                "hostID": 1000,
+                "containerID": 0,
+                "size": 32000
```

Comment:
On Wed, Apr 20, 2016 at 07:23:00PM -0700, Qiang Huang wrote:
> > @@ -463,6 +449,20 @@ Here is a full example `config.json` for reference.
> >                  "gid": 0
> >              }
> >          ],
> > +        "uidMappings": [
> > +            {
> > +                "hostID": 1000,
> > +                "containerID": 0,
> > +                "size": 32000
> 
> 32000 looks like a non sense magic number, 32768 might be better, what do you think?

This PR is just about moving the existing example to the right place,
not about adjusting the example content.

If you want to file a separate issue/PR adjusting the number, that
size is from #370 based on @mrunalp's “32k” suggestion [1].  I'm happy
with anything > 0 and still well short of overflowuid.

[1]: https://github.com/opencontainers/runtime-spec/pull/370#discussion_r58783031